crates/cargo-remove/src/cargo/ops/cargo_remove/mod.rs

```diff
@@ -17,9 +17,9 @@ pub struct RemoveOptions<'a> {
     /// Configuration information for Cargo operations
     pub config: &'a Config,
     /// Package to remove dependencies from
-    pub spec: &'a Package,
```

Comment:
Sorry, I was trying to say it was fine to still borrow this.  It was dependencies that I was wanting to be owned.

Reply:
reverted